endpoints/routing/handler.go

```diff
@@ -37,6 +38,7 @@ func addReadRoutes(cfg config.Configuration, dataStore backends.Backend, appMetr
 	router.GET("/", endpoints.NewIndexHandler(cfg.IndexResponse)) // Default route handler
 	router.GET("/status", endpoints.Status)                       // Determines whether the server is ready for more traffic.
 	router.GET("/cache", endpoints.NewGetHandler(dataStore, appMetrics, cfg.RequestLimits.AllowSettingKeys))
+	router.Handler("GET", "/version", endpoints.NewVersionEndpoint(version.Ver, version.Rev))
```

Comment:
Why use `.Handler` instead of `.GET` like the other routes are using?

Reply:
Corrected to keep consistency